autowarefoundation/autoware
# [Feature] Autoware Launcher

## Status
**DEVELOPMENT**

## Description
This is a experimental release of Autoware Launcher (#1637).

## Todos

## Steps to Test or Reproduce
See Quick Start in [README.md](/CPFL/Autoware/blob/feature/autoware_launcher/ros/src/util/packages/autoware_launcher/README.md)
